buildchain/buildchain/packaging.py

```diff
@@ -328,8 +327,8 @@ def _rpm_package(name: str, sources: List[Path]) -> targets.RPMPackage:
 # All packages not referenced in `RPM_TO_BUILD` but listed in
 # `versions.RPM_PACKAGES` are supposed to be downloaded.
 RPM_TO_DOWNLOAD : FrozenSet[str] = frozenset(
-    "{p.name}-{p.version}-{p.release}".format(p=package)
-    for package in versions.RPM_PACKAGES
+    package.rpm_full_name
+    for package in versions.PACKAGES
```

Comment:
Why is `RPM_PACKAGES` renamed `PACKAGES`?

Reply:
Gah, stray rebase conflict resolution, thank you